app/assets/stylesheets/application.scss

```diff
@@ -3,6 +3,11 @@
 $govuk-use-legacy-palette: true;
 $govuk-typography-use-rem: false;
 
+// govuk_frontend_toolkit dependencies
```

Comment:
Obviously a big ask, but do we still need these imports from the toolkit? This seems like a good opportunity to remove this dependency.

If it's a big job, can we look at it in a separate PR?

Reply:
The tickets for this work are all "do the minimal to bring in the V3 code" with an extra ticket for each to do the tidying up of that kind. The reason being we need to get all the frontend apps into a stable state so that people can adopt new components/update compoents they're using the the apps. The main scope for us of this work is to get the focus states up to date - all the other stuff is nice-to-have but not strictly part of our mission and the progress we are making is really slow, so no, this can't be done immediately but we hope to get to a point where we can.